sal_interfaces/MTM1M3TS/MTM1M3TS_Events.xml

```diff
@@ -1,7 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsl"?>
 <SALEventSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsd">
-  <Enumeration>DetailedStates_DisabledState,DetailedStates_EnabledState,DetailedStates_FaultState,DetailedStates_OfflineState,DetailedStates_StandbyState</Enumeration>
+  <Enumeration>
+    ILCType_ElectromechanicalActuator=1,
+    ILCType_PneumaticActuator=2,
+    ILCType_Thermal=3,
+    ILCType_HardpointMonitoringa=7,
```

Comment:
Thank you for these enums.

Is the "a" at the end of `HardpointMonitoringa` a typo?

Reply:
Yes, coming from vim commands.